### DISCUSSION TITLE
"Layer missing" when restoring VITS model

### DISCUSSION THREAD

**Discussion by madey83** [idiap/coqui-ai-TTS#157] [Q&A] [answered]
hello,

i try to use pre-trained vits female voice to create my own model but when i start process and opened tensorboard i got only this:

![image](https://github.com/user-attachments/assets/6488fa59-3e44-45e8-b3af-5f56846e5f20)

During normal training process tensorboard show "images" "audio" etc...

is this an issue or i done something wrong....

this is command i have used to start process:

D:\TTS-Training\Vits-PL-Model-training> python train_vits_win_Thorsten-Voice.py --restore_path D:\TTS-Training\tts_models--pl--mai_female--vits\model_file.pth.tar --coqpit.run_name "Vits-TTS-PL-finetune"

and also i have got lots below messages:

 > Restoring from model_file.pth.tar ...
 > Restoring Model...
 > Partial model initialization...
 | > Layer missing in the model definition: emb_g.weight
 | > Layer missing in the model definition: emb_l.weight
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.0.weight_g
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.0.weight_v
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.1.weight_g
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.1.weight_v
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.2.weight_g
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.2.weight_v
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.3.weight_g
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.3.weight_v
 | > Layer missing in the model definition: posterior_encoder.enc.in_layers.4.weight_g

**Comment by eginhard** [comment, marked as answer]
It might take some time for the other information to be generated on Tensorboard. The pretrained Coqui models generally only include layers needed for inference, not the ones for training (e.g. the Vits discriminator). So the "Layer missing" messages are expected if starting from those models.

**Comment by madey83** [comment]
thx :)
